src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const normalPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * normalPrice - smallDiscount;
+  }
+
+  return days * normalPrice - bigDiscount;
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Save same parts to const